flow360/component/simulation/models/material.py

```diff
@@ -81,9 +82,13 @@ def prandtl_number(self) -> pd.PositiveFloat:
         return 0.72
 
     @pd.validate_call
-    def speed_of_sound_from_temperature(
-        self, temperature: TemperatureType.Positive
-    ) -> VelocityType.Positive:
+    def pressure(
+        self, density: DensityType.Positive, temperature: TemperatureType.Positive
+    ) -> PressureType.Positive:
+        return density * self.gas_constant * temperature
+
+    @pd.validate_call
+    def speed_of_sound(self, temperature: TemperatureType.Positive) -> VelocityType.Positive:
```

Comment:
This should belong to material because this is an ideal gas relationship. In the future when we add other type's of gas they will have different implementations